app/models/form/sales/questions/person_gender_identity.rb

```diff
@@ -6,6 +6,12 @@ def initialize(id, hsh, page, person_index:)
     @type = "radio"
     @answer_options = ANSWER_OPTIONS
     @check_answers_card_number = person_index
+    @inferred_check_answers_value = [{
+      "condition" => {
+        id => "R",
```

Comment:
Just so I can understand this - what is id here? Thanks!

Reply:
Ah, so id is the id of this question. Because we're reusing this question for all the people, the id might be different each time. 
What this says is that if this question is answered with "R", show "Prefers not to say" in the check your answers instead of the actual answer text

Reply:
Thanks, makes sense! :)